business/MetadataDatabase/Data/SeriesDto.cs

```diff
@@ -25,6 +25,28 @@ public class SeriesDto
         public string StudyID { get; set; }
         public string SeriesNumber { get; set; }
         public string NumberOfSeriesRelatedInstances { get; set; }
+        public string BodyPartExamined { get; set; }
 
+        public void Update(SeriesDto seriesDto)
+        {
+            if (this.SeriesInstanceUID == null) this.SeriesInstanceUID = seriesDto.SeriesInstanceUID;
```

Comment:
This will impact the behaviour of the PUT Rest api, won't it?
No log if the this.field is not null but different form seriesDto.field ?

Reply:
No it is used only for pacs mirror